src/plugins/guided_onboarding/public/components/guide_panel_step.tsx

```diff
@@ -34,6 +35,25 @@ interface GuideStepProps {
   telemetryGuideId: string;
 }
 
+const renderDescription = (description: string | StepDescriptionWithLink) => {
+  if (typeof description === 'string') {
+    return description;
+  }
+  const { descriptionText, linkText, linkUrl, isLinkExternal } = description;
+  return (
+    <>
+      {descriptionText}{' '}
+      <EuiLink
+        data-test-subj="guidePanelStepDescriptionLink"
+        href={linkUrl}
+        external={isLinkExternal}
```

Comment:
[nit] I think by setting `target="_blank"`, `external` will default to `true`, so not sure if this line is needed.

https://elastic.github.io/eui/#/navigation/link#external-links